src/components/notifications/notifications.module.css

```
@@ -0,0 +1,24 @@
+.button {
+    box-shadow: 0px 3px 18px rgba(33, 33, 33, 0.08);
+    border-radius: 8px;
+    min-width: 191px;
+    height: 52px;
+    padding: 18px 49px 17px 18px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.icon {
+    width: 28px;
+    height: 28px;
+    margin-right: 18px;
+    fill: #FFFFFF;
+}
+
+.label {
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 17px;
+    color: #FFFFFF;
+}
```

```suggestion
}

.warning {
   background-color: var(--warning-color);
}

.success {
   background-color: var(--success-color);
}

.error {
   background-color: var(--error-color);
}

```

це стилі `styles[type]` які ми засосуємо в компоненті Nitification 
`clsx(styles.button, styles[type], className)`